corehq/sql_accessors/sql_templates/compare_server_client_case_transaction_order.sql

```diff
@@ -0,0 +1,22 @@
+DROP FUNCTION IF EXISTS compare_server_client_case_transaction_order(TEXT);
+
+CREATE FUNCTION compare_server_client_case_transaction_order(
+    _case_id TEXT
+) RETURNS BOOLEAN AS $$
+BEGIN
+    RETURN (
+        WITH transaction_orders AS (
```

Comment:
For the reason you gave (very small numbers), I don't think it should matter too much. I used it mainly because `with` tends to be much clearer than subqueries to read.

Also, that link seems to say that it's only different (in that case at least) when using explain

Reply:
> that link seems to say that it's only different (in that case at least) when using explain

True, although I would not generalize that outcome. It's hard to know if you'll get the same results unless you know your data is very similar to theirs in terms of row structure, number of rows, etc.